scripts/userdata.sh

```diff
@@ -1,17 +1,86 @@
 #!/bin/bash
 
-DEBUG=$${DEBUG:=off}
+# Install AWS SSM agent for different OS versions
+os_type=$(uname -m)
+os_release=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
 
-if [ $${DEBUG} == 'on' ]; then
-    set -x
+if [ "$${os_type}" == "x86_64" ]; then
+    if [ "$${os_release}" == "amzn" ]; then
+        wget https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
+        sudo rpm --install amazon-ssm-agent.rpm
+        sudo yum update -y
+        sudo yum install -y awslogs vim jq unzip wget tree git
```

Comment:
Do we need `vim jq unzip wget tree git`?

I might have added there because I was using this for a clint in the past.

Reply:
Removed